app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Post;
+use App\Models\User;
+use Auth;
+use Illuminate\Http\Request;
+use App\Http\Requests\PostRequest;
+
+class PostController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $users = User::withCount('posts')->get();
+
+        return view('home', compact('users'));
+    }
+
+    public function show($id)
+    {
+        $user = User::findOrFail($id)->load('posts');
+
+        return view('posts.show', compact('user'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('posts.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(PostRequest $request)
+    {
+        $post = new Post;
+        $name = $request->name;
+        $user_id = Auth::id();
+        $result = $post->create([
+            'name' => $name,
+            'user_id' => $user_id
+        ]);
+        if ($result) {
+            return redirect()->route('posts.create')->with('mess',
+                trans('message.post_create_success'));
+        } else {
+            return redirect()->route('posts.create')->with('mess',
+                trans('message.poss_create_fail'));
+        }
```

Comment:
```suggestion
        if ($result) {
            return redirect()->route('posts.create')->with('mess',
                trans('message.post_create_success'));
        }
            
        return redirect()->route('posts.create')->with('mess', trans('message.poss_create_fail'));
```
sửa những chố tương tự nhé

Reply:
Em đã sửa phần này cho cleancode hơn rồi ạ
```php
  if ($result) {
            return redirect()->route('posts.create')->with('mess',
                trans('message.post_create_success'));
        }

        return redirect()->route('posts.create')->with('mess',
            trans('message.poss_create_fail'));
```